ingestion/src/metadata/ingestion/source/database/databricks/query_parser.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         self.metadata_config = metadata_config
         self.metadata = OpenMetadata(metadata_config)
         self.connection = self.config.serviceConnection.__root__.config
+        self.service_connection = self.config.serviceConnection.__root__.config
```

Comment:
are we using this field anywhere?

Reply:
Yes, it is required,  [here](https://github.com/open-metadata/OpenMetadata/blob/8894b9adf54b0c6412daa2c486d22123e0aca6f1/ingestion/src/metadata/ingestion/source/database/lineage_source.py#L97) otherwise it's giving `AttributeError: 'DatabricksLineageSource' object has no attribute 'service_connection'` Error.